### PR TITLE
Case.ViewCreator - Fixed ViewCreator not launching for Revit 2016,2017,2018

### PR DESCRIPTION
**Pull request by sixtysecondrevit**
Changed Entry namespace to launch correctly. Also hid the about button
on run.

### CODE REVIEW

**Comment by sixtysecondrevit** [comment]
cc @kmorin @rudderdon

**Comment by kmorin** [comment]
Everything looks pretty good; I would fix the spacing though to be consistent and I don't agree with removing 2015 from installer (yet)

**Comment by sixtysecondrevit** [comment]
Cool, I can try to make the changes to 2015 as well, (Currently do not have it installed).

File: Case.AppsRibbon.2017/Entry/AppMain.vb [hunk 1]
```diff
@@ -9,41 +9,41 @@ Imports [Case].AppsRibbon.Utility
 
 Namespace Entry
 
-  ''' <summary>
-  ''' Revit Ribbon Implementation
-  ''' </summary>
-  ''' <remarks></remarks>
-  <Transaction(TransactionMode.Manual)>
-  Public Class AppMain
+    ''' <summary>
```
Comment (kmorin) [inline Case.AppsRibbon.2017/Entry/AppMain.vb]: This is the spacing issues I was referring to. The 2 vs. 4 space indents

File: 2015/Case.ViewCreator/Case.ViewCreator/Case.ViewCreator.csproj [hunk 2]
```diff
@@ -63,11 +63,11 @@
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="RevitAPI">
-      <HintPath>C:\Program Files\Autodesk\Revit 2015\RevitAPI.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\..\ProgramData\Autodesk\ApplicationPlugins\Twinmotion.bundle\Contents\2015\RevitAPI.dll</HintPath>
```
Comment (kmorin) [inline 2015/Case.ViewCreator/Case.ViewCreator/Case.ViewCreator.csproj]: uh oh. what happened here.

**Comment by kmorin** [comment]
Nice work, just need to update that spacing with 2 instead of 4 and it should be good 2 go

File: 2015/Case.ViewCreator/Case.ViewCreator/Case.ViewCreator.csproj (hunk 2 above)
Comment (sixtysecondrevit) [reply 2015/Case.ViewCreator/Case.ViewCreator/Case.ViewCreator.csproj]: Sorry about that. Reference API from wrong folder. Fixed..

**Comment by sixtysecondrevit** [comment]
Awesome, think I got it now. Sorry about that dudes. _Learning is fun, right?_  😄

**Comment by sixtysecondrevit** [comment]
Thanks guys. I really appreciate it. Also, thank you all for building these apps in the first place. I am learning a lot from them. I hope to contribute more in the future.

**Comment by kmorin** [comment]
Thanks for taking some time to contribute!

**Pull request merged**